battery-status-icon.html

```diff
@@ -215,6 +253,17 @@
       return level + "%";
     },
     /**
+    * Converts to human readable time
+    */
+    convertTime: function (totalSeconds) {
+      if ((!totalSeconds) || (totalSeconds == "Infinity") || (totalSeconds == 0)) {
+        return '...';
+      }
+      var hours   = Math.floor(totalSeconds / 3600),
+          minutes = (totalSeconds % 3600) / 60;
```

Comment:
Math.floor() method is missing for minute calculation. Currently the minutes are returned with decimal.